### DISCUSSION TITLE
Will multi-language support be added?

### DISCUSSION THREAD

**Discussion by mtr-static-official** [vercel/vercel#4821] [Ideas]
Hello! Here is a user of Vercel. I think: Vercel users should not be hindered by language barriers. I really hope that Vercel will add multilingual support. If possible, I can provide a Chinese (simplified) translation. Thank you for your attention to my suggestion. Thank you.

**Comment by iinfin** [comment]
I'd love to see this! I could contribute some bits of Chinese as well.
